### DISCUSSION TITLE
Waiting for Kubernetes API

### DISCUSSION THREAD

**Discussion by lruppe** [rancher-sandbox/rancher-desktop#7490] [Q&A] [answered]
Hi guys, I cannot get passed this status while setting up Rancher Desktop with WSL on Windows. Here are the logs, can someone help?

[logs.zip](https://github.com/user-attachments/files/17013561/logs.zip)

**Comment by mook-as** [comment, marked as answer]
Do you have a [`~/.wslconfig`](https://learn.microsoft.com/en-us/windows/wsl/wsl-config#main-wsl-settings) set up with `networkingMode=mirrored`? Rancher Desktop currently does not support that; see #6665.

**Comment by lruppe** [comment]
hey! yes, thanks, that actually solved the issue, many thanks!
